go-controller/pkg/annotation/node.go

```diff
@@ -0,0 +1,34 @@
+package annotation
+
+import (
+	"fmt"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/factory"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+func EnsureNetworkID(watchFactory factory.NodeWatchFactory, nInfo util.NetInfo, networkID *int) error {
```

Comment:
can we just return the value of networkID and not the pointer and take first node available for the ID calc, you should break as soon as you find it from 1st node

Reply:
ok, will leave the value check at controllers.